src/zsys.c

```diff
@@ -981,7 +981,7 @@ zsys_udp_new (bool routable)
     assert (!routable);
     SOCKET udpsock;
     int type = SOCK_DGRAM;
-#if defined ZMQ_HAVE_SOCK_CLOEXEC
+#if defined (SOCK_CLOEXEC)
```

Comment:
@vyskocilm this will breaks some platforms - it needs to be done in cmake and autoconf like in libzmq